packages/core/lib/commands/test/run.js

```diff
@@ -92,6 +127,16 @@ module.exports = async function (options) {
         instamine: "strict"
       }
     };
+    const numberOfFailures = await startGanache(
+      ipcOptions,
+      ganacheOptions,
+      config
+    );
+    return numberOfFailures;
+  }
+
+  // Start internal ganache network
+  async function startGanache(ipcOptions, ganacheOptions, config) {
```

Comment:
Maybe rename this function to `StartGanacheAndRunTests` is a better name for this function?

Reply:
Sure, this name will make more sense.